local-cli/runIOS/RunIOSInternals.js

```diff
@@ -0,0 +1,272 @@
+/**
+* Copyright (c) 2015-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+'use strict';
+
+const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const findXcodeProject = require('./findXcodeProject');
+const parseIOSDevicesList = require('./parseIOSDevicesList');
+const findMatchingSimulator = require('./findMatchingSimulator');
+
+function RunIOSInternals(os_device,
```

Comment:
Can we change it from function arguments to an object and use `flow` to type it?